### PR TITLE
fix #2536

### LINKED ISSUE #2536: Please add an option -> pass text to gallery image alt attribute
https://github.com/gradio-app/gradio/blob/5a090de94a31533667d7d7d0c080a90104744b9e/ui/packages/app/src/components/Gallery/Gallery.svelte#L193

```
def get_images(jpg_path):
    images = []
    for jpg_image in os.listdir(f"{jpg_path}"):
      try:
        images.append((f"{jpg_path}\\{jpg_image}", f"{jpg_image}"))
      except Exception as e:
        print(f"{jpg_path}\\{jpg_image}", e)
    return images
```

```
gallery = gr.Gallery(label=gallery_label, show_label=True, elem_id=f"ats-gallery-{prompt_path}-{jpg_path}").style(grid=[5], height="auto")
btn.click(get_images, txt, gallery)
```

I can pass image label like this ` images.append((f"{jpg_path}\\{jpg_image}", f"{jpg_image}"))` but not image alt please add an option and I think img alt is super important for data science and seo "thank you for coming to my ted talk" 👏👏👏

full code here https://github.com/camenduru/stable-diffusion-webui-artists-to-study

### PR DESCRIPTION
**Pull request by camenduru**
thanks @pngwn

Closes: #2536

### CODE REVIEW

**Comment by camenduru** [comment]
I don't know how svelte works is it compiling now? can I get the compiled code from ci?

File: ui/packages/app/src/components/Gallery/Gallery.svelte [hunk 1]
```diff
@@ -189,11 +189,19 @@
 							class="gallery-item group"
 							on:click={() => (selected_image = can_zoom ? i : selected_image)}
 						>
-							<img
-								alt=""
-								class="h-full w-full overflow-hidden object-contain"
-								src={typeof image === "string" ? image : image.data}
-							/>
```
Comment (pngwn) [inline ui/packages/app/src/components/Gallery/Gallery.svelte:196]: We don't need to duplicate this block and wrap it in an `if`, we can just fall back to an empty string:

```svelte
<img
  alt={caption || ""}
  class="h-full w-full overflow-hidden object-contain"
  src={typeof image === "string" ? image : image.data}
/>
```

**Review by pngwn — CHANGES_REQUESTED** [review]

File: ui/packages/app/src/components/Gallery/Gallery.svelte (hunk 1 above)
Comment (camenduru) [reply ui/packages/app/src/components/Gallery/Gallery.svelte:196]: cool thanks ♥

**Review by pngwn — APPROVED** [review]

**Comment by pngwn** [comment]
Awesome. Thanks so much @camenduru.

Not sure why this is skipping the deploy to spaces workflow but I'll take a look at that.

**Comment by pngwn** [comment]
We just need to add a changelog entry for this. Do you want to take care of that or would you like me to do it?

This would considered a bug fix.

**Comment by freddyaboulton** [comment]
That workflow doesn't run on forks because secrets aren't copied to forks. Need to host that workflow on a separate repo in order to get it to run on forks I think. Something we should do

**Comment by pngwn** [comment]
Not sure what you mean about getting compiled files from CI but you can get everything locally form this branch. The templates are in `gradio/templates` after you have build the frontend by following the instructions in the readme file in `ui/README.md`

**Comment by camenduru** [comment]
You are right, thanks for your help.

**Comment by pngwn** [comment]
This PR is still good, no need to close.

**Comment by pngwn** [comment]
I'll take of the changelog + merge soon.

**Pull request merged**